### PR TITLE
Removes Hugboxing(Man Up edition)(Silo un-maker)

### PR DESCRIPTION
**Pull request by Absolutely-Steph**
## About The Pull Request
This PR removes the Silo from the currently played maps, as well as comments out all occurences of the silo's circuitboard to prevent it from being built entirely

## Why It's Good For The Game
**Best case:** 

- It will return use to the request consoles and make cargo techies do their job of delivering and prioritising.

**Worst case:**

- People will walk to the Ore Redemption machine every time they want mats, and complain to the cargotechies that someone else took all the good stuff. In this case at least they will have to share and will hopefully quickly arrive at the correct answer: letting the cargotechs do their damn job.

Man up, And deal with it.

## Changelog
:cl:
del: Removed ore silos, use the request consoles and let the cargonians do their job.
del: Removed ore silo circuits too.
/:cl:

### CODE REVIEW

**Comment by Evie7056** [comment]
Make sure that RCD silo upgrade can work without silo or it'll need to be killed too.

**Comment by Absolutely-Steph** [comment]
> Make sure that RCD silo upgrade can work without silo or it'll need to be killed too.

right yeah i'll just rip that out too

**Comment by Archemagus** [comment]
I think - better option is to make something with Request Consoles.

You know you can make shortcuts for emergency calling
like
Harm+click = Security Emergency
Disarm+click = Medical Emergency
Grab+click = Engineering Emergency

Help+click = Regular interaction


More over. Request assistance? You kidding? Why you ever need that button if you can shout on common freq "Miners where my hecking diamonds?"

So for now Request Consoles are deprecated and useless, which is really sad.
But by removing features you can't make them great again.

**Comment by Absolutely-Steph** [comment]
> You know you can make shortcuts for emergency calling
> like
> Harm+click = Security Emergency
> Disarm+click = Medical Emergency
> Grab+click = Engineering Emergency


I like that, it's out of scope for this PR, but I will probably add this in a future one.

**Comment by bluey89962** [comment]
Alright, now that I have a little break I can properly write this reply.
_TL;DR: I do not see how, in our current state, removing the Ore Silo benefits the players or the server in any way. To me it's just yet another "The players are lazy and need to learn their lesson >:(" type PR which adds an incovenience for seemingly no reason. We also don't have enough pop for it to be viable._

Let's be real: NSV has been, for a decent while, a lowpop server. Most rounds have 10 people if we're lucky -  and to top it off, our gameplay loop requires much more teamwork amongst the crew than stations do, especially when you consider than in stations, cooperation between the crew is mostly to make cool things and improve things that come round-start, while on NSV it's a matter of not getting blown to pieces by syndicate fleets. This makes it so certain jobs (or someone filling that role) are **necessary** for the ship to even function so people can play for more than 20 minutes. Two people to fly the ship, a few to build and load torp, a miner, an engineer, a doctor, so on and so forth. Of course we have maps that require less people and you can definitely get something done with just three people, but that requires you to do much more than just what your job is supposed to (and that is not necessarily a bad thing - RP wise yes, but you end up learning how to deal with many different situations and how to be the most efficient possible).

We have just the right ammount of people _willing_ to play those jobs so that most rounds will at least run. Players are already (in most rounds) stretched thin - adding a new job (distributing ores around) that is required for the functioning of the ship will just be annoying at best. As crazy as this may sound, not everyone plays NSV to just load torp for five rounds straight, and forcing people to pick yet another job (that they will definitely see as being incredibly boring) to ensure they can even play will just make it so they play a single round, check the crew manifest on the next one to see no one on it and that they will be the ones to do it again, and leave. Removing the ORM will not help lowpop, it will just intensify it. 

Now, onto cargo techs and their duties.

Cargo techs already have more things to do here than they do in normal stations - you have to always keep up with munitions to make sure they're properly stocked and deliver a bunch of crates to them which can be slightly overwhelming when done alone sometimes in addition to doing normal cargo tech stuff.

_"But there's the mail system! It's easy to deliver things!"_
Yes I am very aware of the mail system and I am sure many people are as well. But packaging, marking, renaming (if you want) and delivering 10 crates to that belt one by one (or two by two if you push and drag them at the same time) is no less boring, and it's even worse in some ships due to their design (I'm looking at you, Tycoon).

Did I also mention bounties? Most cargo techs from my experience seem to forget that these exist - And no, just printing 5 bounty papers and pinning them to airlocks is not enough to tell the QM you did your job. Most times instead they just nab the MAA's budget card and use it to buy munitions without worrying about making money. You want cargo techs to work like cargo techs? Find a way so making money is something necessary and not just infinite budget card go brrrr, in my view it's a much better way to insert cargo into the gameplay loop without annoying literally every other department. (this is also good because bounties require cooperation from all departments most times. The bartender can't make all drinks without chemistry, chemistry can't make everything without RnD, and so on. Interdepartamental cooperation good.)

Also I don't even want to think about how many times crates with ores would get lost and just cause a general shitfest. You know how players are. Also having to run back and forth dragging crates isn't what I'd call a fun and engaging experience.

Don't get me wrong - removing the ORM will definitely result in cargo cooperating with other departments more, however we are simply not in the position to do such thing. Though I still don't think it's a good idea in any case.

I am very aware this is badly written but I am currently waiting for my next class to start so I just wanted to post before I forgot about it, but that's my general opinion about it mostly from a player POV since I think it's what matters the most.

**Review by francinum — DISMISSED** [review]
Based.

**Comment by Absolutely-Steph** [comment]
> 



> Alright, now that I have a little break I can properly write this reply. _TL;DR: I do not see how, in our current state, removing the Ore Silo benefits the players or the server in any way. To me it's just yet another "The players are lazy and need to learn their lesson >:(" type PR which adds an incovenience for seemingly no reason. We also don't have enough pop for it to be viable._
> 
> Let's be real: NSV has been, for a decent while, a lowpop server. Most rounds have 10 people if we're lucky - and to top it off, our gameplay loop requires much more teamwork amongst the crew than stations do, especially when you consider than in stations, cooperation between the crew is mostly to make cool things and improve things that come round-start, while on NSV it's a matter of not getting blown to pieces by syndicate fleets. This makes it so certain jobs (or someone filling that role) are **necessary** for the ship to even function so people can play for more than 20 minutes. Two people to fly the ship, a few to build and load torp, a miner, an engineer, a doctor, so on and so forth. Of course we have maps that require less people and you can definitely get something done with just three people, but that requires you to do much more than just what your job is supposed to (and that is not necessarily a bad thing - RP wise yes, but you end up learning how to deal with many different situations and how to be the most efficient possible).
> 
> We have just the right ammount of people _willing_ to play those jobs so that most rounds will at least run. Players are already (in most rounds) stretched thin - adding a new job (distributing ores around) that is required for the functioning of the ship will just be annoying at best. As crazy as this may sound, not everyone plays NSV to just load torp for five rounds straight, and forcing people to pick yet another job (that they will definitely see as being incredibly boring) to ensure they can even play will just make it so they play a single round, check the crew manifest on the next one to see no one on it and that they will be the ones to do it again, and leave. Removing the ORM will not help lowpop, it will just intensify it.
> 
> Now, onto cargo techs and their duties.
> 
> Cargo techs already have more things to do here than they do in normal stations - you have to always keep up with munitions to make sure they're properly stocked and deliver a bunch of crates to them which can be slightly overwhelming when done alone sometimes in addition to doing normal cargo tech stuff.
> 
> _"But there's the mail system! It's easy to deliver things!"_ Yes I am very aware of the mail system and I am sure many people are as well. But packaging, marking, renaming (if you want) and delivering 10 crates to that belt one by one (or two by two if you push and drag them at the same time) is no less boring, and it's even worse in some ships due to their design (I'm looking at you, Tycoon).
> 
> Did I also mention bounties? Most cargo techs from my experience seem to forget that these exist - And no, just printing 5 bounty papers and pinning them to airlocks is not enough to tell the QM you did your job. Most times instead they just nab the MAA's budget card and use it to buy munitions without worrying about making money. You want cargo techs to work like cargo techs? Find a way so making money is something necessary and not just infinite budget card go brrrr, in my view it's a much better way to insert cargo into the gameplay loop without annoying literally every other department. (this is also good because bounties require cooperation from all departments most times. The bartender can't make all drinks without chemistry, chemistry can't make everything without RnD, and so on. Interdepartamental cooperation good.)
> 
> Also I don't even want to think about how many times crates with ores would get lost and just cause a general shitfest. You know how players are. Also having to run back and forth dragging crates isn't what I'd call a fun and engaging experience.
> 
> Don't get me wrong - removing the ORM will definitely result in cargo cooperating with other departments more, however we are simply not in the position to do such thing. Though I still don't think it's a good idea in any case.
> 
> I am very aware this is badly written but I am currently waiting for my next class to start so I just wanted to post before I forgot about it, but that's my general opinion about it mostly from a player POV since I think it's what matters the most.

we're not removing the ORM though.

I've heard many people say we'd be better off without silo, and this is a test to find out if this is true. I'll happily revert it after a month or so if we notice that it's in-fact not a good change (as this is a mass-mapchange, I'd prefer this to not be testmerged for such a long time since making mapping PRs on a testmerged map is cancer.)

File: _maps/map_files/Shrike/job_changes.dm [hunk 1]
```diff
@@ -29,13 +29,13 @@ MAP_REMOVE_JOB(air_traffic_controller)
...
 //Disabled objects
-
+/*NSV13 Ore Silo Removal:
 /obj/item/circuitboard/machine/ore_silo/New()
 	.=..()
 	if(SSmapping?.config?.map_name != JOB_MODIFICATION_MAP_NAME)
 		return
 	build_path = null
-
+*/
```
Comment (Bokkiewokkie) [inline _maps/map_files/Shrike/job_changes.dm:38]: You can just remove this, the Shrike is literally one of our own maps

**Review by Bokkiewokkie — APPROVED** [review]
I feel like if we do this it would make sense to move the job of fabricating parts to Cargo as well, though I won't ask this of you as a requirement.

**Comment by Absolutely-Steph** [comment]
> I feel like if we do this it would make sense to move the job of fabricating parts to Cargo as well, though I won't ask this of you as a requirement.

How would you suggest that to be done? adding an omni-lathe to cargo and removing all the others?

**Comment by Absolutely-Steph** [comment]
I believe this is just about ready, We can experiment with cargo-fabrication later, that seems like a ship-specific thing though.

**Comment by checkraisefold** [comment]
> I think - better option is to make something with Request Consoles.
> 
> You know you can make shortcuts for emergency calling like Harm+click = Security Emergency Disarm+click = Medical Emergency Grab+click = Engineering Emergency
> 
> Help+click = Regular interaction (maybe rework them in some kind S.O.S. terminals)

this is a bad idea, so many people are going to accidentally initiate emergencies, or accidentally bring up a dialogue box and have to click 'no' to initiating one. it should just be an option in the GUI.

**Comment by Archemagus** [comment]
> > I think - better option is to make something with Request Consoles.
> > You know you can make shortcuts for emergency calling like Harm+click = Security Emergency Disarm+click = Medical Emergency Grab+click = Engineering Emergency
> > Help+click = Regular interaction (maybe rework them in some kind S.O.S. terminals)
> 
> this is a bad idea, so many people are going to accidentally initiate emergencies, or accidentally bring up a dialogue box and have to click 'no' to initiating one. it should just be an option in the GUI.

It was just suggestion. It can be realized as radial menu for example. Like Security Hailers.
The main thing that i tried to say - Request Consoles are useless and need total reconsideration of their purpose.

P.S.
when is the last time you used Telecom's Traffic console to send fake or funny PDA messages to crew? Never?
It's same category problem.
(RC reminds me that you can see both RCs and PDAs message logs via that console. Do you ever knew about it?)

**Comment by bluey89962** [comment]
> we're not removing the ORM though.
> 
> I've heard many people say we'd be better off without silo, and this is a test to find out if this is true. I'll happily revert it after a month or so if we notice that it's in-fact not a good change (as this is a mass-mapchange, I'd prefer this to not be testmerged for such a long time since making mapping PRs on a testmerged map is cancer.)

yes I keep typing ORM instead of silo but my point is still the same.
also lets be honest here, most of the people who say we're better without the silo are admins, who usually don't play as much as you know, players do. Knowing how things are, most players would absolutely not be happy with this change. Admin me doesn't care at all about this change and it would be amusing to watch, but player me would definitely be annoyed by having yet another middleman to deal with for something which would add no depth to the round at all.
A testmerge is fine, but I'd rather not have this fully merged if the players **clearly** don't like it, because in that case it really would feel like something that got merged out of spite as I said in the TL;DR in my first reply, because I do not see how this would help "RP" or "pop" in any way shape or form.

**Comment by williamcll** [comment]
Awful, shrike proves how awful this is. Low pop makes this even harder because nobody has time to do material delivery.

**Comment by Kenionatus** [comment]
From my experience on Paradise (that doesn't have an ore silo) departments just go to the orm to collect resources.

**Comment by realroyalderg** [comment]
This is destroying a QOL issue aka the ore silo. You're now encouraging hoarding of materials and less interaction. Instead of everyone having the same pool of materials now nobody has any materials and you better pounce on the mats before they are gone in case you wanna do anything. 

You're also as someone commented before turning a person's job on a relatively low pop server (which isn't a bad thing just a fact) into a crate dragger. So now someone's whole job is to run around a station to act as the ore silo, if you don't have someone that does this then you have mini hordes of materials build up.. which literally detracts from inter-departmental co-operation and can make rounds EVEN longer than they already are sometimes.

I hope this gets reverted quicker than a month and nobody ever touches it ever again

**Comment by Terra1** [comment]
This is a dreadfully bad idea on all accounts, and the person who came up with it should be laughed at for as long as it takes for them to admit their mistake in having the audacity and mistaken confidence to remove a feature so vital to the basic functions of the game.

**Comment by Absolutely-Steph** [comment]
I grew my space legs on early bee and paradise, without ore silo, which worked out fine. I do not particularly care for this change, but it was brought up often enough by the people in charge that I thought it might be a good idea to PR.

Besides that, there has been no greater feeling of pleasure in my recent life than watching the sheer amount of salt emanating from this PR. Some of your reactions here are in such stark contrast to what I have seen in game itself that I am beginning to understand some of the viewpoints expressed by oranges/ike.

> the person who came up with it should be laughed at for as long as it takes for them to admit their mistake in having the audacity and mistaken confidence to remove a feature so vital to the basic functions of the game.

For something so utterly "vital to the basic functions of the game", the crew seem to do just fine without it on the rounds I have spectated until now. The new meta seems to be the construction of a shrike-esque fab studio of sorts near the ORM, which reaffirms bokkie's idea of just having cargo manage it all from the start by default. That should probably be done in a later PR though.

**Comment by Kenionatus** [comment]
> bokkie's idea of just having cargo manage [fabrication] from the start by default.

I like that idea. It gives cargo the option to be proactive in a more interesting way (deciding which department needs what at that moment) than transporting raw resources. When someone comes to request something, they can do paperwork (either by default to add some RP or to get dangerous items approved by a head).

Maybe it's just my Paradise influence where everything researched is printed by RnD, There it comes with the issue that the desk isn't always manned because they don't have to. Cargo is more likely to take that job seriously imo. Even then I think it should be part of cargo SOP that they *have* to process requests. Maybe even something encouraging proactive fabrication? That maybe just in the guide and not SOP?

**Comment by williamcll** [comment]
> I grew my space legs on early bee and paradise, without ore silo, which worked out fine. I do not particularly care for this change, but it was brought up often enough by the people in charge that I thought it might be a good idea to PR.
> 
> Besides that, there has been no greater feeling of pleasure in my recent life than watching the sheer amount of salt emanating from this PR. Some of your reactions here are in such stark contrast to what I have seen in game itself that I am beginning to understand some of the viewpoints expressed by oranges/ike.
> 
> > the person who came up with it should be laughed at for as long as it takes for them to admit their mistake in having the audacity and mistaken confidence to remove a feature so vital to the basic functions of the game.
> 
> For something so utterly "vital to the basic functions of the game", the crew seem to do just fine without it on the rounds I have spectated until now. The new meta seems to be the construction of a shrike-esque fab studio of sorts near the ORM, which reaffirms bokkie's idea of just having cargo manage it all from the start by default. That should probably be done in a later PR though.

we have a round where the CE and HoS just downright called shuttle and cryoed because everything became far more inaccessible for just a diamond cargo run.


Cargo techs are not fun on NSV, forcing people to play an unfun role is disincentive.

Doesn't help that dwindling playercount means cargo is unstaff most of the time the whole department gets tided every round.

**Comment by realroyalderg** [comment]
> I grew my space legs on early bee and paradise, without ore silo, which worked out fine. I do not particularly care for this change, but it was brought up often enough by the people in charge that I thought it might be a good idea to PR.
> 
> Besides that, there has been no greater feeling of pleasure in my recent life than watching the sheer amount of salt emanating from this PR. Some of your reactions here are in such stark contrast to what I have seen in game itself that I am beginning to understand some of the viewpoints expressed by oranges/ike.
> 
> > the person who came up with it should be laughed at for as long as it takes for them to admit their mistake in having the audacity and mistaken confidence to remove a feature so vital to the basic functions of the game.
> 
> For something so utterly "vital to the basic functions of the game", the crew seem to do just fine without it on the rounds I have spectated until now. The new meta seems to be the construction of a shrike-esque fab studio of sorts near the ORM, which reaffirms bokkie's idea of just having cargo manage it all from the start by default. That should probably be done in a later PR though.

Another coder that wants to make things worse and likes salt.. jeez how original.

**Comment by bluey89962** [comment]
> This PR is now part 1 of a series of PRs overhauling cargo as a whole, aiming to bring it closer into our style and moving it from a civilian oriented cargo desk to a fully fledged requisitions department featuring it's own armoury, that will come later, definitely after the new SOP is in.

I do not think it's a very wish idea to turn this specific PR into the "cargo rework PR", especially when it's something that still hasn't even left the ranks of being just an idea thrown around. Not only that but removing the silo especifically is a very controversial change and would definitely make some players less likely to support it if the silo removal is chained to it. If anything I think it'd be wiser to split it into other (future) PRs if it gets going.

**Comment by bluey89962** [comment]
> Yes, that is what I said. This is part of a series of PRs. I will continue with the cargo overhaul whether this gets merged or not, but it will definitely be in a different PR.

Ah, fair, I misunderstood it.

**Comment by Bokkiewokkie** [comment]
Closing this PR because it's clear to me this isn't something that will improve the game on its own right now, and the changes should (if they are made) be part of a PR with a larger scope to improve the game as a whole.
Also to all of you who said walking to and from cargo is a dumb pointless job: It is, that's why you should try using [all the tools at your _disposal._](https://nsv.beestation13.com/wiki/Cargo_Technician#The_Delivery_System)
